app/models/post.rb

```diff
@@ -1,3 +1,14 @@
-# rubocop:disable all
 class Post < ApplicationRecord
+  has_many :comments, class_name: 'Comment', foreign_key: :post_id
+  has_many :likes, class_name: 'Like', foreign_key: :post_id
+
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+
+  def update_post_count
+    author.increment!(:posts_counter)
+  end
+
+  def recent_comments
+    Comment.order(created_at: :desc).limit(5)
+  end
 end
```

Comment:
This is a suggestion it would be nice if you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_post_counter method every time a new post has been saved to database, then make the update_post_counter method private.